apis/keda/v1alpha1/condition_types.go

```diff
@@ -39,6 +39,8 @@ const (
 	ScaledObjectConditionReadySucccesReason = "ScaledObjectReady"
 	// ScaledObjectConditionReadySuccessMessage defines the default Message for correct ScaledObject
 	ScaledObjectConditionReadySuccessMessage = "ScaledObject is defined correctly and is ready for scaling"
+	// ScaledObjectConditionPausedMessage defines the default Message for paused ScaledObject
+	ScaledObjectConditionPausedMessage = "ScaledObject is paused"
```

Comment:
I was thinking about adding a new Condition, not just message.

Reply:
I now added a new Paused condition. Please have a look